### PR TITLE
Feature/248242 tab tile

### PR DESCRIPTION
**Pull request by ms-Danny**

### CODE REVIEW

File: components/tab-tile/tab-tile.scss [hunk 1]
```diff
@@ -12,26 +18,74 @@
...
+  &__text {
+    max-width: 200px;
+  }
+
+  &:hover {
+    color: $clear-blue;
+  }
+
+  &:focus {
+    outline: 4px solid rgba(0, 153, 218, 0.5);
```
Comment (mattsskea) [inline components/tab-tile/tab-tile.scss]: This rgba(0, 153, 218, 0.5) has been showing up throughout the design system, maybe we should look into moving rgba colours into the _variables.scss file

File: components/tab-tile/tab-tile.js [hunk 2]
```diff
@@ -1,20 +1,77 @@
 const tabs = document.querySelectorAll('.msds-tab-tile')
+const tabContainers = document.querySelectorAll('.row-scroll-x-mobile')
+const viewport = window.matchMedia('(max-width: 992px)')
```
Comment (mmussmann) [inline components/tab-tile/tab-tile.js]: it's a media query list not a viewport

File: components/tab-tile/tab-tile.js [hunk 3]
```diff
@@ -1,20 +1,77 @@
...
+  const elementWidth = tab.clientWidth
+  const calcTabScrollPos = tabScrollLeft - containerWidth / 2 + elementWidth / 1.5
+
+  const scrollOptions = {
+    left: calcTabScrollPos,
+    top: 0,
+    behavior: 'smooth'
+  }
+
+  parentElement.scrollTo(scrollOptions)
```
Comment (mmussmann) [inline components/tab-tile/tab-tile.js]: do you have a fallback for edge and IE?

File: components/tab-tile/tab-tile.scss (hunk 1 above)
Comment (mmussmann) [reply components/tab-tile/tab-tile.scss]: discuss with Martin what he thinks

File: components/tab-tile/tab-tile.scss [hunk 4]
```diff
@@ -12,26 +18,74 @@
...
+    border: 1px solid $gray-4;
     background: $white;
     border-top: 4px solid $clear-blue;
     border-bottom: none;
+    color: $clear-blue;
+  }
+
+  &--disabled {
+    color: $gray-7;
+    filter: grayscale(1);
```
Comment (mmussmann) [inline components/tab-tile/tab-tile.scss]: doesn't look to have widespread support?
https://caniuse.com/#search=grayscale

Comment (ms-Danny) [reply components/tab-tile/tab-tile.scss]: No IE 11 is not really happy with the grayscale, ive added correct colours to all other elements on the tab, so CE will have to supply us with the  correct image when building the block

File: components/tab-tile/tab-tile.js (hunk 3 above)
Comment (mmussmann) [reply components/tab-tile/tab-tile.js]: Make sure it doesn't throw console errors

File: components/tab-tile/tab-tile.js [hunk 5]
```diff
@@ -21,12 +21,19 @@ for (let i = 0; i < tabs.length; i++) {
...
-    tabs.forEach(tab => {
+    for (let i = 0; i < tabs.length; i++) {
+      const tab = tabs[i]
       const tabHasFocus = document.activeElement
       if (tabHasFocus === tab) {
         setActive(tab)
       }
-    })
+    }
+    // tabs.forEach(tab => {
```
Comment (mmussmann) [inline components/tab-tile/tab-tile.js]: remove comments

File: components/tab-tile/tab-tile.js [hunk 6]
```diff
@@ -38,12 +45,20 @@ function setActive(clickedTab) {
...
+  for (let i = 0; i < tabs.length; i++) {
+    const tab = tabs[i]
     const isActive = tab.classList.contains(activeClass)
     if (isActive) {
       tab.classList.remove(activeClass)
     }
-  })
+  }
+
+  // tabs.forEach(tab => {
```
Comment (mmussmann) [inline components/tab-tile/tab-tile.js]: remove comments

**Pull request merged**